src/transformers/modeling_rope_utils.py

```diff
@@ -216,14 +271,17 @@ def _compute_dynamic_ntk_parameters(
         Tuple of (`torch.Tensor`, `float`), containing the inverse frequencies for the RoPE embeddings and the
         post-processing scaling factor applied to the computed cos/sin (unused in this type of RoPE).
     """
-    # TODO (joao): use the new `original_max_position_embeddings` from rope_scaling
-    base = config.rope_theta
-    partial_rotary_factor = getattr(config, "partial_rotary_factor", 1.0)
+    # TODO (joao): use the new `original_max_position_embeddings` from rope_parameters
+    # For backward compatibility standardize the `rope_parameters_dict` if it uses old format
+    standardize_rope_params(config)
```

Comment:
this was already called in the config's init we should not need to call it twice